README.md

```diff
@@ -4,21 +4,26 @@ HostName Plugin for RUTorrent
 The purpose of this plugin is to show hostnames of peers for selected torrent.
 Based on GeoIP plugin.
 
-![Screenshot](screenshot.png)
-
+![Screenshot](https://i.imgur.com/VrPA3O2.png)
```

Comment:
I already have: http://www.netlab.linkpc.net/download/software/rtorrent/rutorrent/hostnames.png

Reply:
IMHO using a well-known image hoster is prefered. =)

Reply:
Well known user tracker :)
I prefer to not use "well-known" to avoid web centralization and for better privacy.

Reply:
Okay, it's your choice. :D